clients/ctl/admin-ui/src/features/dataset/DatasetCollectionView.tsx

```diff
@@ -23,16 +25,28 @@ import { ColumnMetadata, EditableType } from "./types";
 const ALL_COLUMNS: ColumnMetadata[] = [
   { name: "Field Name", attribute: "name" },
   { name: "Description", attribute: "description" },
-  { name: "Personal Data Categories", attribute: "data_categories" },
   { name: "Identifiability", attribute: "data_qualifier" },
+  { name: "Personal Data Categories", attribute: "data_categories" },
 ];
 
-const useDataset = (key: string) => {
-  const { data, isLoading } = useGetDatasetByKeyQuery(key);
+/**
+ * Query subscriptions shared between vies within this feature.
+ */
+const useSubscriptions = (key: string) => {
+  const { data: dataset, isLoading: isDatasetLoading } =
+    useGetDatasetByKeyQuery(key);
+  const features = useFeatures();
+  const { isLoading: isClassificationsLoading } = useGetAllClassificationsQuery(
+    undefined,
+    {
+      skip: !features.plus,
```

Comment:
it only just occurred to me that this could be a handy way around not putting hooks in if statements... I feel like I'm always running into that problem

Reply:
omg right? I always imagine an alternate reality where hooks accepted a key/id argument instead of relying on consistent order.